### ISSUE TITLE
Lesson update: Create Block Theme plugin

### ISSUE THREAD

**Issue by jonathanbossenger** [WordPress/Learn#2514]
This is a new issue to track a minor update to an existing lesson #2197 

https://github.com/WordPress/create-block-theme/pull/595

It looks like CBT no longer has the font management User Interface, relying instead on the built-in core Font library.

All that is needed is to edit the existing Manage your Fonts section (from around 2:10 to about 3:07), and the rest of the video is still valid.

We can revisit this once the rest of the learning pathway is complete.

**Comment by jonathanbossenger** [comment]
@cynthianorman I'm creating this as a separate "update" issue for #2197, so we can keep that original issue closed. The reason for this is that the closed issues in GitHub are used to track published lessons, so it looks like this issue is still in progress, when in fact it's published, an only needs a minor update. 

I'm going to add this to an "Todos" section on the original learning pathway tracking issue.

**Comment by cynthianorman** [comment]
ok @jonathanbossenger this is now in my todo list - thank you for separating it out

**Comment by jonathanbossenger** [comment]
BTW No rush here. If it comes to it, I can also help with editing once I wrap up my last theme developer module.

**Comment by jonathanbossenger** [comment]
https://wordpress.tv/2024/04/26/create-block-theme-plugin/
https://www.youtube.com/watch?v=d8nNf9suqxI
